lib/humanized_id/humanizer.rb

```diff
@@ -0,0 +1,43 @@
+require 'facets'
+
+module HumanizedId
+  class Humanizer
+    def initialize(id:, length: nil, prefix: '')
+      @id               = id
+      @length           = length || id.to_s.length
+      @prefix           = prefix
```

Comment:
If someone passes `prefix: nil` this won't actually set it to the empty string.
However, as it's only used in the string interpolation it works fine.